Fix android example

umarniz opened the pull request:
This extends the changes introduced by #282 and makes them work also on Android 9+ devices.

This has been tested to work correctly on an Android 9 device.

slycoder commented on Example/index.js:
```diff
@@ -98,7 +99,10 @@ export default class Example extends Component {
   }
 
-  _onConnectButtonPress = () => {
+  _onConnectButtonPress = async () => {
```
Does there need to be an if Platform.OS === 'android' around these awaits?

umarniz commented:
Thanks for the review @slycoder!

I have fixed based on your feedback and also added another function which I forgot to commit.

Tested to work on a brand new cloned repo on Android + iOS 🚀
